src/test/java/Repo/OrderRepoTest.java

```diff
@@ -0,0 +1,59 @@
+package Repo;
+
+import modul1.Order;
+import modul1.Product;
+import org.testng.annotations.Test;
```

Comment:
Vorsicht, hier ist die "Test" Annotation aus TestNg importiert. Hier sollte der Import von junit rein :) 
```suggestion
import org.junit.jupiter.api.Test;
```